streams/pe.m3u

```diff
@@ -155,6 +155,8 @@ http://cdnhd.iblups.com/hls/F87ppt1YAT.m3u8
 http://ott.streann.com:8080/loadbalancer/services/public/channels/59ce7f292cdc7ba015a93b82/playlist.m3u8
 #EXTINF:-1 tvg-id="LaRiberena.pe" status="error",La Ribereña (Pucallpa) (480p) [Not 24/7]
 https://inliveserver.com:1936/19020/19020/playlist.m3u8
+#EXTINF:-1 tvg-id="Latina.pe",Latina (720p)
+http://free.fullspeed.tv/iptv-query?streaming-ip=https://www.latina.pe/tvenvivo
```

Comment:
Not working at the moment:

<img width="729" alt="image" src="https://user-images.githubusercontent.com/7253922/159168021-2920a9b9-ee18-4fc9-9489-966adf890ba4.png">

Reply:
Yes, because I don't think @dct-infra pushed the last Streamlink changes to his repo yet.